base/memory/memory_pressure_monitor_endless.cc

```diff
@@ -116,10 +116,16 @@ int MemoryPressureMonitor::GetUsedMemoryInPercent() {
   // Since swappable memory uses a non pre-deterministic compression and
   // the compression creates its own "dynamic" in the system, it gets
   // de-emphasized by the |kSwapWeight| factor.
-  const int kSwapWeight = 4;
+  //
+  // Endless: Since it's not possible to know in advance how much this value
+  // will be in practice, we're using the one that we've observed so far since
+  // we started enabling zram-backed swap partitions.
+  const int kSwapWeight = 2.7;
```

Comment:
We could add a command line switch for it, something like `--swap-compression-ratio=<value>`, and I think that's a really good idea, so I'll tinker a bit with it to see if I can come up with something that does the job without taking me forever (otherwise I'll leave it for a separate PR).

Reply:
Actually, one more thing: this `kSwapWeight` should be a `double`!!

Reply:
Both things fixed. Also, I added an additional column in the "Memory" section of chrome://discards, so that we can easily check whether the value passed via the cmdline switch is actually in use.